Code/Core/FileIO/FileIO.cpp

```diff
@@ -71,6 +72,23 @@
     } gOSXHelper_utimensat;
 #endif
 
+/*static*/ FileIO::SymlinkBehavior FileIO::ParseSymlinkBehavior( const AString& value )
+{
+	if ( value.IsEmpty() || value.CompareI( "FollowSrcAndDest" ) )
```

Comment:
This is where to change the default behavior of CopyFile in bff code.  If you want it to match the new way and not the normal way, you'd have IsEmpty return `FOLLOW_NEITHER`

Reply:
Also welcome any feedback on these names.  Are there other examples of reflected "enums"?